### PR TITLE
Adding new credential management docs

### PR DESCRIPTION
**Pull request by Meggin**
I'm going to do a round of bug fixes, but wanted to get this out for review as soon as possible.

### CODE REVIEW

**Comment by agektmr** [comment]
Actually, I would rather put this series of documents under `src/content/en/fundamentals/getting-started/primers/credential-management-api/`. The other documents we worked on was designed to link to these.
@Meggin What do you think? If you agree, please change.

File: src/content/en/fundamentals/security/credential-management/index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+{# wf_updated_on: 2016-11-08 #}
+{# wf_published_on: 2016-11-08 #}
+
+# The Credential Management API {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+{% include "web/_shared/contributors/megginkearney.html" %}
+
+Add the following features to your site using the
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/index.md]: Need stronger intro here

File: src/content/en/fundamentals/security/credential-management/index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  session has expired.
+* **Disable auto sign-in:** Once a user signs out, disable automatic sign-in
+  for the user's next visit.
+
+Note: This API must be used on secure origins on top level context.
+
+## User sign-in flow
+
+{% comment %}
+Watch the user sign-in flow for a
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/index.md]: Confused by this

File: src/content/en/fundamentals/security/credential-management/index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+1. Retrieve user credentials.
+2. Save user credentials.
+3. Disable auto sign-in when user signs out.
+
+### Retrieve user credentials
+
+To retrieve user credentials:
+
+1. When a user lands on your site and no signed in sessions are available,
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/index.md]: no signed in sessions - where, need more of a definition of what this means

File: src/content/en/fundamentals/security/credential-management/index.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+1. Retrieve user credentials.
+2. Save user credentials.
+3. Disable auto sign-in when user signs out.
+
+### Retrieve user credentials
+
+To retrieve user credentials:
+
+1. When a user lands on your site and no signed in sessions are available, 
+   call `navigator.credential.get()` with `unmediated: true`.
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/index.md]: need to define what `unmediated` means somewhere

File: src/content/en/fundamentals/security/credential-management/retrieve-credentials.md [hunk 5]
```diff
@@ -0,0 +1,371 @@
...
+
+{# wf_updated_on: 2016-11-08 #}
+{# wf_published_on: 2016-11-08 #}
+
+# Retrieve Credentials {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+{% include "web/_shared/contributors/megginkearney.html" %}
+
+When a user lands on your site, retrieve the user's credentials. Use these
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/retrieve-credentials.md]: Need stronger intro here

File: src/content/en/fundamentals/security/credential-management/retrieve-credentials.md [hunk 6]
```diff
@@ -0,0 +1,371 @@
...
+
+`navigator.credentials.get()` returns a promise that resolves with a
+credential object as an argument. The obtained credential object can be either
+[`PasswordCredential`](#authenticate_with_a_server) or
+[`FederatedCredential`](#authenticate_with_an_identity_provider). If no
+credential information exists, `null` gets returned.
+
+### Parameters
+
+A credential includes the following parameters:
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/retrieve-credentials.md]: Are these the params for `navigator.credentials.get()`

File: src/content/en/fundamentals/security/credential-management/retrieve-credentials.md [hunk 7]
```diff
@@ -0,0 +1,371 @@
...
+      </td>
+      <td>
+        <code>String</code><br>
+        User icon image url string.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Change parameters
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/retrieve-credentials.md]: What parameters, why would I change this?

File: src/content/en/fundamentals/security/credential-management/retrieve-credentials.md [hunk 8]
```diff
@@ -0,0 +1,371 @@
...
+
+Change the param keys by assigning a string to `.idName` or `.passwordName`.
+
+You can also add extra parameters such as a CSRF token by assigning a
+`FormData` as `.additionalData` property and append key-values to it.
+
+For example:
+
+    }).then(function(cred) {
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/retrieve-credentials.md]: the `}).then(` looks uber confusing, how can we simplify

File: src/content/en/fundamentals/security/credential-management/retrieve-credentials.md [hunk 9]
```diff
@@ -0,0 +1,371 @@
...
+
+* The user has not acknowledged the automatic sign-in feature (once per
+  browser instance).
+* The user has either no credentials or more than two credential objects
+  stored on the origin.
+* The user has requested to require user mediation to the origin.
+
+## Sign in a user via account chooser
+
+If a user requires mediation, or has multiple accounts, use the account
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/retrieve-credentials.md]: What's mediation?

File: src/content/en/fundamentals/security/credential-management/retrieve-credentials.md [hunk 10]
```diff
@@ -0,0 +1,371 @@
...
+navigator.credentials.get({
+  password: true,
+  <strong>unmediated: true,</strong> // request a credential without user mediation
+  federated: {
+    providers: [
+      'https://account.google.com',
+      'https://www.facebook.com'
+    ]
+  }
+}).then(function(cred) {
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/retrieve-credentials.md]: Can we drop this, it's not used

File: src/content/en/fundamentals/security/credential-management/store-credentials.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+
+{# wf_updated_on: 2016-11-08 #}
+{# wf_published_on: 2016-11-08 #}
+
+# Store Credentials {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+{% include "web/_shared/contributors/megginkearney.html" %}
+
+Keep credential information always sane and up-to-date in the password
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/store-credentials.md]: Need better intro here

File: src/content/en/fundamentals/security/credential-management/store-credentials.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+<div class="clearfix"></div>
+
+## Store a credential from a form element
+
+Mapping form input values into a `PasswordCredential` is non-trivial.
+
+In order to do so, you must explicitly annotate input fields in a form and
+then instantiate one directly from the form element.
+
+Anotate the input fields, `id` and `password`, using
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/store-credentials.md]: Can we point to our own docs for this?

File: src/content/en/fundamentals/security/credential-management/store-credentials.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+    var form = document.querySelector('#form');
+    var cred = new PasswordCredential(form);
+    // Store it
+    navigator.credentials.store(cred)
+    .then(function() {
+      // continuation
+    });
+
+
+Note: `.additionalData` also gets added automatically.
```
Comment (petele) [inline src/content/en/fundamentals/security/credential-management/store-credentials.md]: Wait, what's `.additionalData`? this is the first i've heard of it.

**Pull request merged**